docs/functions.json

```diff
@@ -1357,6 +1357,51 @@
             }
         ]
     },
+    {
+        "signature": "trim(text, [,chars])",
+        "summary": "Removes whitespace from either end of `text`.",
+        "detail": "There is an optional final parameter `chars` which is string of characters to be removed instead of whitespace.",
+        "examples": [
+            {
+                "template": "@(trim(\" hello world    \"))",
+                "output": "hello world"
+            },
+            {
+                "template": "@(trim(\"+123157568\", \"+\"))",
+                "output": "123157568"
+            }
+        ]
+    },
+    {
+        "signature": "trim_left(text, [,chars])",
```

Comment:
was also wondering if `trim_start` and `trim_end` might be nicer for non-Go programmers.. but no strong feelings